client/app/components/queries/visualization-embed.html

```diff
@@ -32,9 +32,9 @@ <h3>
           <span am-time-ago="$ctrl.queryResult.getUpdatedAt()" ng-if="!$ctrl.loading"></span>
           <rd-timer from="$ctrl.refreshStartedAt" ng-if="$ctrl.loading"></rd-timer>
         </a>
+        <span class="small visible-print"><i class="zmdi zmdi-time-restore"></i> {{$ctrl.queryResult.getUpdatedAt() | dateTime}} UTC</span>
```

Comment:
Please give this `margin-left: 5px` so it's not so close to the timeago text.

Reply:
What element should receive the extra margin?

Reply:
```suggestion
        <span class="small visible-print m-l-5"><i class="zmdi zmdi-time-restore"></i> {{$ctrl.queryResult.getUpdatedAt() | dateTime}} UTC</span>
```

Reply:
But this shows up only when the other is hidden or I am misunderstanding something?

Reply:
Nope it's me who misunderstood how this works.